bicycleparameters/main.py

```diff
@@ -479,7 +479,7 @@ def add_rider(self, riderName, reCalc=False, draw=False):
             self.riderName = riderName
             self.hasRider = True
 
-    def plot_bicycle_geometry(self, show=True, pendulum=True,
+    def plot_bicycle_geometry(self, show=False, pendulum=True,
```

Comment:
This should not be changed in main, but you can change it when you call the function.